CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
  - #1530, Fix how the PostgREST version is shown in the help text when the `.git` directory is not available - @monacoremo
 
+### Changed
+
+ - Docker images are now built from scratch based on a static build of the PostgREST executable.
```

Comment:
```suggestion
 - #1535, Docker images are now built from scratch based on a static build of the PostgREST executable.
```
Also, maybe mention that the image is optimized now(from 30 mb to 4mb).

It'll be good to merge after this.